Source/stores.cpp

```diff
@@ -805,18 +852,19 @@ void S_StartNoRoom()
 
 void S_StartConfirm()
 {
-	bool idprint;
-
 	StartStore(stextshold);
 	stextscrl = false;
 	ClearSText(5, 23);
-	uint16_t iclr = plr[myplr].HoldItem.getTextColor();
 
-	idprint = plr[myplr].HoldItem._iMagical != ITEM_QUALITY_NORMAL;
+	auto &item = plr[myplr].HoldItem;
+
+	uint16_t iclr =item.getTextColor();

```

Comment:
Some whitespace got removed here.
```suggestion
	uint16_t iclr = item.getTextColor();
```